test/hotspot/jtreg/vmTestbase/nsk/share/jpda/DebugeeProcess.java

```diff
@@ -84,6 +85,10 @@ abstract public class DebugeeProcess extends FinalizableObject {
     protected DebugeeProcess (DebugeeBinder binder) {
         this.binder = binder;
         this.log = binder.getLog();
+
+        // As the alternative to finalize(), register the cleanup() method
+        // to be called when this instance becomes unreachable.
+        Cleaner.create().register(this, () -> cleanup());
```

Comment:
Why do we need to do this explicitly here? Why not call `registerCleaner`?

Reply:
Was remained from the former solution. `registerCleanup()` is called.